apps/web/test/utils/bookingScenario/expects.ts

```diff
@@ -1192,6 +1192,7 @@ export function expectBookingPaymentIntiatedWebhookToHaveBeenFired({
 
 type ExpectedForSuccessfulCalendarEventCreationInCalendar = {
   calendarId?: string | null;
+  calendarIdUsingFallbackOfFirstCalendarCredential?: string | null;
```

Comment:
Added the capability to be able to test the fallback flow through which event is created on a calendar